Gopkg.toml

```diff
@@ -62,11 +62,6 @@
   name = "github.com/Masterminds/semver"
   version = "v1.3.1"
 
-[[override]]
-  name = "gopkg.in/yaml.v2"
-  source = "https://github.com/simonferquel/yaml"
```

Comment:
FWIW @simonferquel don't remove your branch, otherwise older release may no longer be able to check vendoring (if you want to get rid of the branch, you could tag it so that it's preserved)